data/css/_colors.scss

```diff
@@ -7,10 +7,3 @@ $accent-dark-color: #cc532b !default;
 
 $background-light-color: #e9e9e3 !default;
 $background-dark-color: #ddddd5 !default;
-
-$white-60-color: transparentize(white, 0.4) !default;
```

Comment:
@ptomato I don't know, I see your point but still think we should get rid of them; I don't think something repeated enough times necessarily means a variable should be made out of it. The fact that these variables represent static values pretty much defeats the purpose of it being a variable at all. Also, 2 things -- in our CSS, we flip all the time between using the variables or using the actual values, and then when we don't use the variables, sometimes we might want another transparency value. I don't think we should have variables for these.

But I do agree that maybe having a `$overlay` color system might be interesting, if we use it enough. But right now, we've only been using these colors to define different transparencies of text colors, and sometimes background colors; we don't use overlays that much. But that seems like a better use case for variables if we did want that in the future.